app/models/immunisation_import.rb

```diff
@@ -24,6 +24,9 @@ class ImmunisationImport < ApplicationRecord
   attr_accessor :csv_is_malformed, :data, :rows
 
   belongs_to :user
+  has_many :vaccination_records,
+           dependent: :nullify,
```

Comment:
Do we definitely want this? I imagine if we tried to delete an immunisation import that had created some vaccination records we'd want to stop it from being deleted to keep a record of it?

Reply:
Yeah, I thought the same, but thought adding something to prevent deletion is more complex/requires some sort of hook

However, I just discovered `:restrict_with_exception` which seems like a better fit here: https://guides.rubyonrails.org/association_basics.html#options-for-has-one-dependent